src/util-napatech.c

```diff
@@ -392,11 +818,19 @@ static uint32_t CountWorkerThreads(void)
     return worker_count;
 }
 
+/**
+ * \brief Reads and parses the stream configuration defined in the config file.
+ *
+ * \param stream_config - array to be filled in with active stream info.
+ *
+ * \return success or failure
```

Comment:
Suggest adding scalar values returned ... other places return `0` and `1` but not here.